src/main/java/Event.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
the same comment goes for the Deadline class. any reason why you put this variable as protected?